src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -7,44 +7,91 @@
 
 import com.revrobotics.CANPIDController;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel;
 import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
 
 import frc.robot.Constants;
 
 public class ShooterSubsystem extends SubsystemBase {
-  private final CANSparkMax ShooterMAX = new CANSparkMax(Constants.SPARKMAX_SHOOTER_CAN_ADDR,
+  /**
+   * Initializes the SparkMAX motor controller, assigns it to the CAN address
+   * specified, and sets it to the NEO Brushless Motor.
+   */
+  private final CANSparkMax shooterMAX = new CANSparkMax(Constants.SPARKMAX_SHOOTER_CAN_ADDR,
       CANSparkMaxLowLevel.MotorType.kBrushless);
-
-  private final CANPIDController ShooterPID = new CANPIDController(ShooterMAX);
-
-  private final Encoder shootEncoder = new Encoder(Constants.SHOOTER_ENCODER_CHAN_A, Constants.SHOOTER_ENCODER_CHAN_B);
-
-  private final double ShooterSetpoint = 1.0;
-
+  /**
+   * The built-in PID controller provided by the Spark MAX motor controller.
+   */
+  private final CANPIDController shooterPID = new CANPIDController(shooterMAX);
+  /**
+   * The target velocity of the NEO Brushless Motor.
+   */
+  private final double shooterSetpoint = 1.0;
+  /**
+   * The Proportial Gain of the SparkMAX PIDF controller The weight of the
+   * proportional path against the differential and integral paths is controlled
+   * by this value.
+   */
   private final double kP = 1;
+  /**
+   * The Integral Gain of the SparkMAX PIDF controller The weight of the integral
+   * path against the proportional and differential paths is controlled by this
+   * value.
+   */
   private final double kI = 1;
+  /**
+   * The Differential Gain of the SparkMAX PIDF controller. The weight of the
+   * differential path against the proportional and integral paths is controlled
+   * by this value.
+   */
   private final double kD = 1;
+  /**
+   * The Integral Zone of the SparkMAX PIDF controller. The integral accumulator
+   * will reset once it hits this value.
+   */
   private final double kIz = 1;
+  /**
+   * The Feed-Forward Gain of the SparkMAX PIDF controller. The weight of the
+   * feed-forward loop as compared to the PID loop is controlled by this value.
+   */
   private final double kFF = 1;
-  private final double OutputRange = 1;
+  /**
+   * Scales the output of the SparkMAX PIDF controller.
+   */
+  private final double outputRange = 1;
 
   public ShooterSubsystem() {
-    ShooterPID.setP(kP, 0);
-    ShooterPID.setI(kI, 0);
-    ShooterPID.setD(kD, 0);
-    ShooterPID.setIZone(kIz, 0);
-    ShooterPID.setFF(kFF, 0);
-    ShooterPID.setOutputRange(-OutputRange, OutputRange, 0);
+    // Applies the previously-declared values to the PIDF controller.
+    shooterPID.setP(kP, 0);
+    shooterPID.setI(kI, 0);
+    shooterPID.setD(kD, 0);
+    shooterPID.setIZone(kIz, 0);
+    shooterPID.setFF(kFF, 0);
+    shooterPID.setOutputRange(-outputRange, outputRange, 0);
```

Comment:
Consider also setting the current limiting feature of the SparkMAX.  NEO should be 40A.